### ISSUE TITLE
Procedure of uprading/degrading games statuses

### ISSUE THREAD

**Issue by mattmaniak** [Whisky-App/whisky-book#114]
From a maintainer perspective I find it a little chaotic that any game status have it's statuses upgraded or degraded ad hoc like here [PR #113](https://github.com/Whisky-App/whisky-book/pull/113). It is not a big issue now but it may become hard to keep all the titles up to date with their statuses or at least have any history of those statuses. We don't set game statuses per machine specs which can be differ from chip to chip or model to model. As an example, I have tested A Plague Tale: Innocence and it worked once but after a fresh installation in a new Bottle it stopped.

I don't have any long term solution for it right now but I think that it may be helpful especially that we are close to reaching over 200 tested and documented titles.

**Comment by IsaacMarovitz** [comment]
The correct solution here would be to use an actual database rather than a static book to hold game support information, as CrossOver does. That would be a fair amount of work, and I would need to ensure it's secure.

**Comment by mattmaniak** [comment]
I agree. The question is infrastructure. We would have to create some kind of backend that will cost some day and keep to maintain it. Is it doable from the funding side or do you prefer to try something else? Some kind of cloud may be the most reliable and efficient solution for us but... it costs from some level.

**Comment by rimvydasb** [comment]
I wonder if such a site hit the free tier limit. vercel.com and Cloudflare Pages offer generous free tiers.
Initially, it would be great to have at least a Google Docs Sheet or Form to make a minimal public contribution. 

I won't clone whisky-book and deal with an editor to inform the world that Jets'n'Guns 2 works well on my Mac. The absence of an acceptable public contribution process holds whisky back.

**Comment by mattmaniak** [comment]
Thanks @rimvydasb for pointing out poor UX for making a contribution. Google Forms sounds easy-to-use. Maybe this way: Google Forms -> Cloud Firestore -> Cloud Compute VM -> (preparing a PR using Steam API to scrap game data) -> (creating PR using GitHub CLI? @IsaacMarovitz Firebase Spark (free) plan sounds fair enough for us.
